### PR TITLE
Make snapshot image compression configurable and other minor tweaks

### PR DESCRIPTION
**Pull request by mandoonandy**

### CODE REVIEW

File: shakenfist/config.py [hunk 1]
```diff
@@ -166,6 +166,10 @@ class SFConfig(BaseSettings):
         1800, description='How long to wait for an image transfer to occur before giving up'
     )
 
+    COMPRESS_SNAPSHOTS: bool = Field(
+        False, description='Compress snapshots taken of instances'
```
Comment (mikalstill) [inline shakenfist/config.py:170]: I don't agree that the default should be false. I think this should default to true and then you can override it in your config file. Compression is the right choice, especially in terms of disk IO performance for new instances in most cases.

**Review by mikalstill — CHANGES_REQUESTED** [review]
Just one small tweak please.

File: shakenfist/config.py (hunk 1 above)
Comment (mandoonandy) [reply shakenfist/config.py]: I knew you wouldn't like that. I do posit that it's very slow and thus makes the user experience horrible.

**Review by mikalstill — APPROVED** [review]
Thanks.

**Pull request merged**